clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/validators/servergroup/DeployOpenstackAtomicOperationValidator.groovy

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright 2016 Veritas Technologies LLC.
+ * Copyright 2016 Target, Inc.
```

Comment:
No - what is our policy when multiple teams make significant changes to a file? 

@lwander any suggestions?